docs/getting-connector.md

```diff
@@ -49,39 +41,35 @@ You can download the Spark-Riak Connector library by going to https://github.com
 Here's another way to download and install it in your home directory:
 
 ```
-curl https://bintray.com/artifact/download/basho/data-platform/com/basho/riak/spark-riak-connector/{{version}}/spark-riak-connector-{{version}}-uber.jar \
-                    -o ~/spark-riak-connector-{{version}}-uber.jar
+curl http://repo1.maven.org/maven2/com/basho/riak/spark-riak-connector_{{scala-version}}/{{connector-version}}/spark-riak-connector_{{scala-version}}-{{connector-version}}-uber.jar \
```

Comment:
We probably want to use `-sS` as options to `curl`. Maven will sometimes fail to resolve things for unexpected reasons and we need to report the error. I also don't like the progress meter that curl prints so I hide that (also makes it easier to use in things like `Dockerfiles`). I'd also recommend `-O` rather than making the command longer by having to provide a `-o <<path>>`.